lib/active_merchant/billing/gateways/omise/omise_vault.rb

```diff
@@ -0,0 +1,37 @@
+require 'active_merchant/billing/gateways/omise/omise_core'
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class OmiseVaultGateway < Gateway
```

Comment:
No need for another gateway here; this should be implemented on the main gateway as `store`.